### PR TITLE
Make codable array Storable

### PR DESCRIPTION
**Pull request by bangtoven**
I had to make this change to store array of `Codable`s.

### CODE REVIEW

File: ios/CBStore/Protocols/Storable.swift [hunk 1]
```diff
@@ -63,3 +63,5 @@ public extension Storable where Self: Codable {
         return try? decoder.decode(self, from: data)
     }
 }
+
+extension Array: Storable where Element: Codable { }
```
Comment (bangtoven) [inline ios/CBStore/Protocols/Storable.swift]: `extension Array: DefaultStorable {}` wasn't enough to support decodable array.

File: ios/StoresTests/StoreTests.swift [hunk 2]
```diff
@@ -59,6 +59,16 @@ class StoreTests: XCTestCase {
 
         XCTAssertEqual(obj, stores.get(.codableKey))
     }
+    
+    func testUserDefaultArrayBasedKey() {
+        let expected = [
+            ExampleStruct(name: "User 1", age: 11),
+            ExampleStruct(name: "User 2", age: 22)
+        ]
```
Comment (AndrewGold) [inline ios/StoresTests/StoreTests.swift]: Nit: The style we typically go with is to put a new line after multi line statements.

Comment (bangtoven) [reply ios/StoresTests/StoreTests.swift]: would that be like this?
```suggestion
        ]

```

Comment (AndrewGold) [reply ios/StoresTests/StoreTests.swift]: Perfect, thanks

**Pull request merged**